pegasus/sites.v3/code.org/views/census_map.haml

```diff
@@ -81,9 +104,56 @@
 
     google.maps.event.addListener(layer, 'click', function(event) {
       enableDrag();
-      event.infoWindowHtml = event.row['organization_name'].value;
-      event.infoWindowHtml += ' (' + event.row['city'].value + ')';
-      event.infoWindowHtml += '<br>';
+      let censusMessage;
+      switch(event.row['teaches_cs'].value) {
+        case 'YES':
+          censusMessage = "We believe this school offers Computer Science class(es).";
+          break;
+        case 'NO':
+          censusMessage = "We believe this school offers limited or no Computer Science opportunities.";
+          break;
+        case 'MAYBE':
+          censusMessage = "We are unsure if this school offers Computer Science class(es).";
+          break;
+        default:
+          censusMessage = "We have no data about this school.";
+      }
+      const schoolDropdownOptionString = `{
+        value: '${event.row['school_id'].value}',
+        label: '${event.row['school_name'].value}',
+        school: {
+          nces_id: '${event.row['school_id'].value}',
+          name: '${event.row['school_name'].value}',
+          city: '${event.row['city'].value}',
+          state: '${event.row['state_code'].value}',
+          latitude: ${event.row['location'].value.split(',')[0]},
+          longitude: ${event.row['location'].value.split(',')[1]},
+        },
+      }`;
+      event.infoWindowHtml  = '<div class="census-info-window">';
+      event.infoWindowHtml += '  <h4>';
```

Comment:
Could a template literal make this a bit more concise?  https://stackoverflow.com/a/805113

Reply:
Yes. I'm made the change.